src/configurations/adknowledgeportal/synapseConfigs/projects.ts

```diff
@@ -68,15 +67,35 @@ export const projectsDetailsPageConfiguration: DetailsPageProps = {
       },
     },
     {
-      name: 'CardContainerLogic',
-      columnName: 'Grant Number',
+      name: 'ToggleSynapseObjects',
       title: 'Experimental Models',
       showTitleSeperator: false,
-      tableSqlKeys: ['grant'],
-      props: {
-        sql: experimentalModelsSql,
-        ...experimentalToolsCardConfiguration,
+      standalone: true,
+      toggleConfigs: {
```

Comment:
toggleConfigs defines the two configs to use, along with what icons (Icon enum defined in IconSvg)